intercom/scroll_collection_proxy.py

```diff
@@ -14,6 +14,9 @@ def __init__(self, client, resource_class, resource_name, scroll_url):
         # resource name
         self.resource_name = resource_name
 
+        # key name of resource list
+        self.resource_key = "data"
```

Comment:
Similar to the change made [here](https://github.com/Twingate/python-intercom/commit/7ffc4b4abf9cc1bceeb32f17a143c7ab971daa5b) once upon a time.